cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -64,6 +64,16 @@ spec:
       labels:
         k8s-app: kube-proxy
     spec:
+      initContainers:
+      - name: touch-lock
+        image: busybox
```

Comment:
The image needs to be templated to be cross arch. For this type of simple command, we usually just use busybox (though I'd like to switch to alpine). E.g. https://github.com/kubernetes/kubernetes/blob/25aed0a63181ab43222ba24b2f090fd1952d8db6/cluster/images/etcd/Makefile#L37